File: SayiOkuma.js
<!-- HTML + JavaScript kullanılmıştır. -->

<html>

<p>Bir sayı giriniz.</p>

<script src="sayioku.js"></script>
<input id="yazialani" type="number">
<button onclick="SayiOkuma()">Sayıyı oku!</button>

</html>

//sayioku.js dosyası:

    function SayiOkuma() {
        document.write("Girdiğiniz sayı: " + document.getElementById('yazialani').value);
    }

File: SayiOkuma.java
package bilgisayarkavramlari;
import java.util.Scanner;
public class BilgisayarKavramlari {
    public static void main(String[] args) {
        Scanner sayiGir = new Scanner(System.in);
        System.out.print("Bir sayi giriniz: ");
        int sayi = sayiGir.nextInt();
        System.out.println("Girdiğiniz sayı: "+sayi);
    }
    
}
